src/Component/Symbolizer/FillEditor/FillEditor.tsx

```diff
@@ -106,6 +108,29 @@ export class FillEditor extends React.Component<FillEditorProps> {
     } = this.props;
     const symbolizer: FillSymbolizer = _cloneDeep(this.props.symbolizer);
     symbolizer.fillOpacity = value;
+    window.console.log('changed:', value);
+    if (onSymbolizerChange) {
+      onSymbolizerChange(symbolizer);
+    }
+  };
+
+  onOpacityChange = (value: number) => {
+    const {
+      onSymbolizerChange
+    } = this.props;
+    const symbolizer: FillSymbolizer = _cloneDeep(this.props.symbolizer);
+    symbolizer.opacity = value;
+    if (onSymbolizerChange) {
+      onSymbolizerChange(symbolizer);
+    }
+  };
+
+  onStrokeOpacityChange = (value: number) => {
```

Comment:
Please rename everything with `stroke` to `outline` to be consistent with the geostyler-style.

Reply:
Done in dae52fc